src/Compilers/Core/Portable/Syntax/SyntaxTrivia.cs

```diff
@@ -194,9 +206,9 @@ public IEnumerable<SyntaxAnnotation> GetAnnotations(params string[] annotationKi
         /// </summary>
         /// <returns>The child non-terminal node representing the syntax tree structure under this structured
         /// trivia.</returns>
-        public SyntaxNode GetStructure()
+        public SyntaxNode? GetStructure()
         {
-            return HasStructure ? UnderlyingNode.GetStructure(this) : null;
+            return HasStructure ? UnderlyingNode!.GetStructure(this) : null;
```

Comment:
>UnderlyingNode! [](start = 34, length = 15)

nit: Consider adding a comment to mark `HasStructure` with `MemberNotNull(nameof(UnderlyingNode))` once we have the capability

Reply:
Good idea. Filed a general issue we can use to track all places like this. 

https://github.com/dotnet/roslyn/issues/41964